front/js/cart.js

```diff
@@ -1,92 +1,324 @@
+    productItemContentSettingsQuantity.appendChild(productQte)
+    productQte.innerHTML = "Qté : "
 
     // Insertion de la quantité
-    let productQuantity = document.createElement("input");
-    productItemContentSettingsQuantity.appendChild(productQuantity);
-    productQuantity.value = produitLocalStorage[product].quantiteProduit;
-    productQuantity.className = "itemQuantity";
-    productQuantity.setAttribute("type", "number");
-    productQuantity.setAttribute("min", "1");
-    productQuantity.setAttribute("max", "100");
-    productQuantity.setAttribute("name", "itemQuantity");
+    let productQtty = document.createElement("input")
+    productItemContentSettingsQuantity.appendChild(productQtty)
+    productQtty.value = productLocalStorage[product].productQuantity
+    productQtty.className = "itemQuantity"
+    productQtty.setAttribute("type", "number")
+    productQtty.setAttribute("min", "1")
+    productQtty.setAttribute("max", "100")
+    productQtty.setAttribute("name", "itemQuantity")
 
     // Insertion de l'élément "div"
-    let productItemContentSettingsDelete = document.createElement("div");
-    productItemContentSettings.appendChild(productItemContentSettingsDelete);
-    productItemContentSettingsDelete.className = "cart__item__content__settings__delete";
+    let productItemContentSettingsDelete = document.createElement("div")
+    productItemContentSettings.appendChild(productItemContentSettingsDelete)
+    productItemContentSettingsDelete.className = "cart__item__content__settings__delete"
 
     // Insertion de "p" supprimer
-    let productSupprimer = document.createElement("p");
-    productItemContentSettingsDelete.appendChild(productSupprimer);
-    productSupprimer.className = "deleteItem";
-    productSupprimer.innerHTML = "Supprimer";
+    let productSupprimer = document.createElement("p")
+    productItemContentSettingsDelete.appendChild(productSupprimer)
+    productSupprimer.className = "deleteItem"
+    productSupprimer.innerHTML = "Supprimer"
 }
 }}
-getCart();
\ No newline at end of file
+getCart()
+
+function getTotals(){
+
+    // Récupération du total des quantités
+    let elementQuantity = document.getElementsByClassName('itemQuantity')
+    let elementLength = elementQuantity.length,
+    totalQtt = 0
+
+    for (let index = 0; index < elementLength; ++index) {
+        totalQtt += elementQuantity[index].valueAsNumber
+    }
+
+    let productTotalQuantity = document.getElementById('totalQuantity')
+    productTotalQuantity.innerHTML = totalQtt
+    console.log(totalQtt)
+
+    // Récupération du prix total
+    totalPrice = 0;
+
+    for (let index = 0; index < elementLength; ++index) {
+        totalPrice += (elementQuantity[index].valueAsNumber * productLocalStorage[index].prodcutPrice)
+    }
+
+    let productTotalPrice = document.getElementById('totalPrice')
+    productTotalPrice.innerHTML = totalPrice
+    console.log(totalPrice)
+}
+getTotals()
+
+function modifyQtt() {
```

Comment:
call you function something else... because that's a function that is gonna register event handlers. so addEventListennerQuantityChange